assessment/services/google_storage.py

```diff
@@ -11,4 +12,7 @@ def setup_google_storage_credentials():
 
 
 def upload_file_to_google_bucket(destination_file_name, bucket_name, file):
-    pass
+    storage_client = storage.Client()
+    bucket = storage_client.get_bucket(bucket_name)
+    blob = bucket.blob(destination_file_name)
+    blob.upload_from_file(file_obj=file, rewind=True)
```

Comment:
Hello, Jo. If I may ask, the functionality of uploading blobs are built in from the Google Cloud package, yes?

Reply:
Hello, Alex. Yes, the blobs are built in from the Google cloud module.

Reply:
Okay thanks Jo. This means I need to install the package when I pull from staging.